hydroshare/hs_macros.py

```diff
@@ -1052,7 +1052,7 @@ class Resource(Hydroshare):
     abstract = SiteElement(By.ID, "id_abstract")
     abstract_save = SiteElement(By.CSS_SELECTOR, "#div_id_abstract button")
     public_resource_notice = SiteElement(By.ID, "missing-metadata-or-file")
-    subject_keywords = SiteElement(By.ID, "txt-keyword")
+    subject_keyword_input = SiteElement(By.ID, "txt-keyword")
```

Comment:
Why was this changed?

Reply:
I wanted to make it more obvious that this targets the input element (which is deeply nested within the Vuetify element) vs the entire subject keywords form element. But you're right this rename isn't necessary, this was important before I removed some of the other `subject keywords...` elements during refactoring